firefox-ios/Client/Frontend/Microsurvey/Survey/View/MicrosurveyTableHeaderView.swift

```diff
@@ -73,8 +69,9 @@ class MicrosurveyTableHeaderView: UITableViewHeaderFooterView, ReusableCell, The
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(_ text: String) {
+    func configure(_ text: String, icon: UIImage?) {
         questionLabel.text = text
+        iconView.image = icon?.withRenderingMode(.alwaysTemplate)
```

Comment:
@adudenamedruby I decided to move the default configuration on the nimbus config side instead in the code for easier configurability.